packages/terra-icon/src/Icon.module.scss

```
@@ -21,3 +21,34 @@
     transform: scaleX(-1);
   }
 }
+
+:global {
+  @keyframes tui-spin-ltr {
+    0% {
+      transform: rotate(0deg);
+    }
+    
+    100% {
+      transform: rotate(359deg);
+    }
+  }
+
+  @keyframes tui-spin-rtl {
+    0% {
+      transform: rotate(0deg);
+    }
+    
+    100% {
+      transform: rotate(-359deg);
+    }
+  }
+
+  /* stylelint-disable selector-class-pattern */
+  [dir=ltr] .tui-Icon.is-spin {
```

You can do this, which will scope the selector but not apply the scope to the animation name
```suggestion
  :local([dir=ltr] .tui-Icon.is-spin) {
```

---

Updated here https://github.com/cerner/terra-core/pull/3187/commits/c51499d2d550c87e7551e34717068ae4a895a861